src/stdatamodels/jwst/datamodels/schemas/emi.schema.yaml

```diff
@@ -0,0 +1,43 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/emi.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_filter.schema
+- $ref: keyword_pfilter.schema
+- $ref: keyword_readpatt.schema
+- $ref: keyword_preadpatt.schema
+- $ref: keyword_psubarray.schema
+- $ref: subarray.schema
+- type: object
+  properties:
+    data:
+      title: MIRI Reference Waves Table for EMI correction
+      type: object
```

Comment:
Will this table be stored in a FITS HDU? If so (or even if its for a structured ndarray that will end up in the ASDF HDU) perhaps [nrsfs_apcorr.schema.yaml](https://github.com/spacetelescope/stdatamodels/blob/master/src/stdatamodels/jwst/datamodels/schemas/nrsfs_apcorr.schema.yaml#L12) can serve as an example. There are a few differences including:
- using [fits_schema](https://github.com/spacetelescope/stdatamodels/blob/4a06de96585dad1c8497e4fd01d4041ee2c210b4/src/stdatamodels/jwst/datamodels/schemas/nrsfs_apcorr.schema.yaml#L3) instead of `asdf-schema` for the `$schema`
- defining a [fits_hdu](https://github.com/spacetelescope/stdatamodels/blob/4a06de96585dad1c8497e4fd01d4041ee2c210b4/src/stdatamodels/jwst/datamodels/schemas/nrsfs_apcorr.schema.yaml#L14) for the table
- defining the [compound datatype](https://github.com/spacetelescope/stdatamodels/blob/4a06de96585dad1c8497e4fd01d4041ee2c210b4/src/stdatamodels/jwst/datamodels/schemas/nrsfs_apcorr.schema.yaml#L15) which defines the table columns
Let me know if it's helpful to make a PR with this changes against the source branch of this PR. I considering adding 'recommendations' here but the changes would likely be confusing given the amount of reorganization.

Reply:
It is to be stored in ASDF format

Reply:
Thanks! Is there an example reference file?

Reply:
Oh! I just read the `table-like` comment in `emi.py`. This looks good for a `table-like` object.